trusted-committer-script.md

```diff
@@ -0,0 +1,57 @@
+# InnerSource Learning Path Script: Trusted Committer
```

Comment:
One thing that I think would be interesting to chat about is the proposed grouping of the content into video(s).  How many segments will there be, about how long will they take, and what will be the goal/points of each one?